### ISSUE TITLE
render fails: multiple entries in ExperimentHub

### ISSUE THREAD

**Issue by vjcitn** [lmweber/BestPracticesST#46]
Quitting from lines  at lines 23-32 [unnamed-chunk-1] (analysis-steps.qmd)
Error in Visium_humanDLPFC() : 
  "Visium_humanDLPFC" matches more than 1 ExperimentHub resource
Calls: .main ... eval_with_user_handlers -> eval -> eval -> Visium_humanDLPFC

I am seeing EH7538 and EH9516, apparently for different bioc versions?
```
> mcols(qq)$rdatapath # qq is output of query(eh, "Visium_humanDLPFC")
[1] "STexampleData/3_15/Visium_humanDLPFC.rds"
[2] "STexampleData/3_19/Visium_humanDLPFC.rds"
```

**Comment by lmweber** [comment]
Thanks for reporting this. Yes, we accidentally duplicated some of the object names in the `STexampleData` package when we updated these before the Bioc release. I have contacted Lori Kern to fix this by removing the older ones, which should remove the duplication. Once we have done this, the error here should be resolved.

If it takes longer, I could also temporarily add an update here to use the EHub IDs instead of the named loader functions, which would also fix the issue for now.
